week-2/extended-project/yoda-y-sus-apis/web_api.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "web_api_methods"
+
+#Get info from apis
+characters = get_data("https://swapi.co/api/people/")
+
+films = get_data("https://swapi.co/api/films/")
+
+planets =  get_data("https://swapi.co/api/planets/")
+
+#Filter the data that we only need
+names = []
```

Comment:
Why did you declare a variable that is never used?

Reply:
Oh thanks, I didn't realize!!